hassio/dock/addon.py

```diff
@@ -100,7 +109,8 @@ def _run(self):
                 ports=self.addons_data.get_ports(self.addon),
                 devices=self.addons_data.get_devices(self.addon),
                 environment=self.environment,
-                volumes=self.volumes
+                volumes=self.volumes,
+                tmpfs=self.tmpfs
```

Comment:
Done. I have modified the vol.Match to be directly compatible with the tmpfs api option